### DISCUSSION TITLE
Put Events request to EventBridge Global EndPoint  Errors with SigV4a signer

### DISCUSSION THREAD

**Discussion by pkaflekafle** [aws/aws-sdk-java-v2#4334] [Q&A]
HI,

I am unable to send Put Events request to EventBridge Global EndPoint and constantly getting same error shown below.   I can send request to EventBus but not to global endpoint using 'software.amazon.awssdk.eventbridge '

I can send request to Endpoint using python  but not with java based code. 
<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 15">
<meta name=Originator content="Microsoft Word 15">
<link rel=File-List
href="file:///C:/Users/PK95825/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<link rel=themeData
href="file:///C:/Users/PK95825/AppData/Local/Temp/msohtmlclip1/01/clip_themedata.thmx">
<link rel=colorSchemeMapping
href="file:///C:/Users/PK95825/AppData/Local/Temp/msohtmlclip1/01/clip_colorschememapping.xml">

<style>

</style>

</head>

<body lang=EN-US style='tab-interval:.5in;word-wrap:break-word'>

<h1 style='margin:0in'><span style='font-size:10.0pt;mso-fareast-font-family:
"Times New Roman";font-weight:normal'>AWS Service:&nbsp; <o:p></o:p></span></h1>

<h1 style='margin:0in'><span style='font-size:10.0pt;mso-fareast-font-family:
"Times New Roman";font-weight:normal'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<span
style='color:#232F3E'>Global endpoints for Amazon EventBridge<o:p></o:p></span></span></h1>

<p class=MsoNormal><span style='font-size:12.0pt'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
Issue with </span><span style='font-size:9.0pt;font-family:Consolas;color:#24292F;
background:white'>software.amazon.awssdk.eventbridge </span><span
style='font-size:12.0pt'><o:p></o:p></span></p>

<java.version>17</java.version>            <aws.java.sdk.version>2.20.115</aws.java.sdk.version>            <springcloud.version>2022.0.4</springcloud.version>
--
  |  

</body>

</html>

**_Error:_**
Cannot find the software.amazon.awssdk.authcrt.signer.AwsCrtV4aSigner class. To invoke a request that requires a SigV4a signer, such as region independent signing, the 'auth-crt' core module must be on the class path. 
at software.amazon.awssdk.auth.signer.SignerLoader.initializeV4aSigner([SignerLoader.java]

**Comment by debora-ito** [comment]
The error message has the answer:

> To invoke a request that requires a SigV4a signer, such as region independent signing, the 'auth-crt' core module must be on the class path.

You need to add `auth-crt` as a dependency.

```
    <dependency>
        <groupId>software.amazon.awssdk</groupId>
        <artifactId>auth-crt</artifactId>
        <version>2.20.132</version>
    </dependency>
```
